app/models/post.rb

```diff
@@ -1,2 +1,15 @@
 class Post < ApplicationRecord
+  has_many :comment
+  has_many :like
+  belongs_to :author, class_name: 'User'
+
+  after_save :update_posts
+
+  def update_posts
+    author.increment!(:posts_counter)
+  end
```

Comment:
- [ OPTIONAL ] Great job for using one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post method every time a new post has been saved to database, but please I recommend making the update_posts method private ✨🚀🔥